x-pack/solutions/security/plugins/security_solution/public/detections/components/alerts_table/translations.ts

```diff
@@ -418,3 +418,10 @@ export const EVENT_RENDERED_VIEW_COLUMNS = {
     defaultMessage: 'Event Summary',
   }),
 };
+
+export const NOT_ENOUGH_PRIVILEGES = i18n.translate(
```

Comment:
what do you think about renaming the const to `INSUFFICIENT_PRIVILEGES` (as well as it's id) to match the renamed text?

Reply:
Oh yeah, honestly I forgot to update the name of the variable after I updated the translation 😂  thanks for catching this!